plugins/modules/katello_content_view_version.py

```diff
@@ -201,18 +202,18 @@ def main():
     if 'current_lifecycle_environment' in entity_dict:
         entity_dict['current_lifecycle_environment'] = module.find_resource_by_name(
             'lifecycle_environments', name=entity_dict['current_lifecycle_environment'], params=scope)
-        search = "content_view_id={0},environment_ids=[{1}]".format(content_view['id'], entity_dict['current_lifecycle_environment']['id'])
-        content_view_version = module.find_resource('content_view_versions', search=search, thin=True)
+        search_scope = {'content_view_id': content_view['id'], 'environment_id': entity_dict['current_lifecycle_environment']['id']}
+        content_view_version = module.find_resource('content_view_versions', search=None, params=search_scope)
```

Comment:
Is there a reason for this change other than the `thin`thing?

Reply:
Yes,  you can't search for `environment_ids=[{1}]`, you need to pass `environment_id` as a param.